.azure/gpu-tests.yml

```
@@ -74,11 +74,11 @@ jobs:
         python -c "fname = 'requirements/pytorch/strategies.txt' ; lines = [line for line in open(fname).readlines() if 'bagua' not in line] ; open(fname, 'w').writelines(lines)"
         TORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
-        CUDA_VERSION_BAGUA=$(python -c "print([ver for ver in [115,113,111,102] if $CUDA_VERSION_MM >= ver][0])")
+        CUDA_VERSION_BAGUA=$(python -c "print([ver for ver in [116,115,113,111,102] if $CUDA_VERSION_MM >= ver][0])")
```

I think we can let 115 go since it was a workaround for 116. cc @otaj
```suggestion
        CUDA_VERSION_BAGUA=$(python -c "print([ver for ver in [116,113,111,102] if $CUDA_VERSION_MM >= ver][0])")
```